src/App.js

```diff
@@ -8,15 +8,55 @@ class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: movieData.movies,
+      movies: [],
       clickedMovie: null,
+      isLoading: false,
+      error: '',
     }
   }
 
-  displayMovieDetails = (id) => {
-    const singleMovie = this.state.movies.find(movie => movie.id === id)
+  componentDidMount = () => {
+    this.setState({isLoading: true})
+
+    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
```

Comment:
Nice work setting up your network request in `componentDidMount`.  A nice refactor could be to move the network request into a `apiCalls.js` file and import it/call it here!